datalad_deprecated/resources/website/index.html

```diff
@@ -29,6 +30,8 @@
 	</noscript>
 	<script type="text/javascript" charset="utf-8" src=".git/datalad/web/assets/js/jquery-2.2.4.js"></script>
 	<script type="text/javascript" charset="utf-8" src=".git/datalad/web/assets/js/jquery.dataTables-1.10.12.js"></script>
+	<script type="text/javascript" charset="utf-8" src="https://cdnjs.cloudflare.com/ajax/libs/jquery-contextmenu/2.9.2/jquery.contextMenu.min.js"></script>
+	<script type="text/javascript" charset="utf-8" src="https://cdnjs.cloudflare.com/ajax/libs/jquery-contextmenu/2.9.2/jquery.ui.position.min.js"></script>
```

Comment:
we should carry and "deploy" copies of those along with the rest of .js.  Hopefully they do not need much of dependencies in turn